core/scripts/local/VoteGasEstimation.js

```diff
@@ -52,60 +53,104 @@ async function run() {
   // Pick a random time. Probabilistically, this won't request a duplicate time from a previous run.
   let time = Math.floor(Math.random() * max);
 
-  console.log("total supply", (await votingToken.totalSupply()).toString());
+  console.log("Voting tokens total supply:", (await votingToken.totalSupply()).toString());
 
   for (var i = 0; i < numRounds; i++) {
-    console.log("Round", i);
+    console.group(`\n*** Round ${i} ***`);
     await cycleRound(voting, votingToken, identifier, time, accounts);
     time += numPriceRequests;
+    console.groupEnd();
   }
 
+  console.group(`\nVoter token balances post-test:`);
   for (var i = 0; i < numVoters; i++) {
     const voter = getVoter(accounts, i);
-    console.log("balance", (await votingToken.balanceOf(voter)).toString());
+    const balance = await votingToken.balanceOf(voter);
+    console.log(`- Voter #${i}: ${balance.toString()}`);
   }
-  console.log("total supply", (await votingToken.totalSupply()).toString());
+  console.groupEnd();
+  console.log("Voting tokens total supply", (await votingToken.totalSupply()).toString());
 
-  console.log("Done");
+  console.groupEnd();
 }
 
+// Cycle through each round--consisting of a price request, commit, and reveal phase
 const cycleRound = async (voting, votingToken, identifier, time, accounts) => {
+  /**
+   * Estimating gas usage: requestPrice
```

Comment:
nit: why the /* style comment? We generally prefer the `//` comment for short, inline code comments.